scipy/stats/tests/common_tests.py

```diff
@@ -312,9 +312,6 @@ def check_freezing(distfn, args):
 def check_rvs_broadcast(distfunc, distname, allargs, shape, shape_only, otype):
     np.random.seed(123)
     with suppress_warnings() as sup:
-        # frechet_l and frechet_r are deprecated, so all their
-        # methods generate DeprecationWarnings.
-        sup.filter(category=DeprecationWarning, message=".*frechet_")
```

Comment:
I think the entire context manager can go. Same for all the others I think.

Reply:
Yeah, if the only warning being suppressed is the frechet deprecation, the context manager can go.  If it helps, you can see where the context managers were added in https://github.com/scipy/scipy/pull/7838/files

Reply:
Good catch, thanks.